pandas/util/testing.py

```diff
@@ -306,12 +306,21 @@ def _skip_if_scipy_0_17():
         pytest.skip("scipy 0.17")
 
 
-def _skip_if_no_lzma():
+def _check_if_lzma():
     try:
         return compat.import_lzma()
     except ImportError:
-        import pytest
-        pytest.skip('need backports.lzma to run')
+        return False
```

Comment:
Can't use it here because there's a 2 vs 3 import in `compat`, though I guess it could be in an `if PY3` instead of `try`/`except` like it it now.